x-pack/test/api_integration/apis/synthetics/add_monitor_project.ts

```diff
@@ -1846,5 +1846,191 @@ export default function ({ getService }: FtrProviderContext) {
         await deleteMonitor(httpProjectMonitors.monitors[1].id, project);
       }
     });
+
+    it('project monitors - handles sending invalid public location', async () => {
+      const project = `test-project-${uuidv4()}`;
+      try {
+        const response = await supertest
+          .put(`${API_URLS.SYNTHETICS_MONITORS_PROJECT_UPDATE.replace('{projectName}', project)}`)
+          .set('kbn-xsrf', 'true')
+          .send({
+            monitors: [
+              {
+                ...httpProjectMonitors.monitors[1],
+                locations: ['does not exist'],
+              },
+            ],
+          })
+          .expect(200);
+        expect(response.body).eql({
+          createdMonitors: [],
+          failedMonitors: [
+            {
+              details:
+                'Invalid location "does not exist" supplied to field "locations". Please select a valid location.',
```

Comment:
```suggestion
                'Invalid location: "does not exist". Remove it or replace it with a valid location.',
```
Is "does not exist" the name of the location here or is it the cause of why it's invalid? 

If this is the cause and not the name, I'd suggest this instead:
`The provided location does not exist. Remove it or replace it with a valid location.`